vendor/bat-native-ads/src/bat/ads/internal/conversions/conversions.cc

```diff
@@ -488,6 +483,10 @@ void Conversions::ProcessQueue() {
 
         ConversionQueueItemInfo conversion_queue_item =
             conversion_queue_items.front();
+        if (conversion_queue_item.ad_type == AdType::kUndefined) {
+          // Migrate legacy conversion queue items
```

Comment:
could this be done from within the migration file? We might not know the ad type then I guess

Reply:
Fixed